stellar-captive-core/debian/control

```diff
@@ -0,0 +1,12 @@
+Source: stellar-captive-core
+Section: web
+Priority: optional
+Maintainer: Package Maintainer <packages@stellar.org>
+Build-Depends: debhelper (>=9)
+Standards-Version: 3.9.6
+Homepage: https://www.stellar.org/
+
+Package: stellar-captive-core
+Architecture: any
+Depends: stellar-core-postgres, stellar-horizon
```

Comment:
Is this right? Does captive-core need PG?

Reply:
captive-core doesn't, but stellar-core (which it depends on) does.  the stellar-core-postgres package is a virtual packages that configures stellar-core and postgres on the same box

Reply:
Stellar-Core running in captive mode does not need access to a DB, it stores all the data in memory. So I think we don't need postgres. @tamirms can confirm.

Reply:
@bartekn is right, the DB is not necessary